android/src/main/java/com/swmansion/gesturehandler/core/RotationGestureHandler.kt

```diff
@@ -59,7 +59,7 @@ class RotationGestureHandler : GestureHandler<RotationGestureHandler>() {
       anchorY = point.y
     }
     if (sourceEvent.actionMasked == MotionEvent.ACTION_UP) {
-      if (state == STATE_ACTIVE) {
+      if (state == STATE_ACTIVE && sourceEvent.pointerCount == 0) {
```

Comment:
Is this change necessary? I think it's the same as with pinch - `pointerCount` may never be equal to 0.